include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3265,12 +3270,14 @@ namespace internal
   template <int dim, int fe_degree, typename Number>
   struct CellwiseInverseMassMatrixImpl
   {
-    template <typename FEEvaluationType>
     static void
-    apply(const unsigned int      n_components,
-          const FEEvaluationType &fe_eval,
-          const Number *          in_array,
-          Number *                out_array)
+    apply(const unsigned int                        n_components,
+          const FEEvaluationMappingRelated<dim,
+                                           typename Number::value_type,
```

Comment:
I guess it is guaranteed that Number is of type VectorizedArray?

Reply:
Currently yes because `FEEvaluation` assumes that the data type is always a `VectorizedArray<Number,width>`. We could relax that as soon as `FEEvaluation` opens for other types.